core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -12,7 +12,7 @@
          ng-class="$ctrl.windowIsNarrow ? 'oppia-logo-small' : 'oppia-logo-wide'" alt="Oppia Home">
   </a>
   <!-- This is needed for the correct image to appear when an exploration is shared using G+. -->
-  <a style="display: none;">
+  <a class="exploration-shared-image-display">
```

Comment:
Rename the class name such that it follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css).

> If the CSS class is oppia-specific, prefix it with oppia-. This helps distinguish it from CSS classes used by other third-party libraries.

Reply:
@DubeySandeep Does that mean all classes that are not third-party have to get prefixed with "oppia-"?

Reply:
Yeah, as documented on wiki page,! :)